src/screens/CreatePost/index.js

```diff
@@ -610,6 +622,7 @@ const CreatePost = () => {
 
   const isPollButtonDisabled = () => getReducedPoll().length < 2;
   const isButtonDisabled = () => {
+    if (loadingPost) return true;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This line checks if a post is currently being created by checking the `loadingPost` state. If a post is being created, the button is disabled. This is a good practice as it prevents users from triggering multiple post creation processes at the same time.
```diff
+    if (loadingPost) return true;
     if (isPollShown) return isPollButtonDisabled();
     return false;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->